### PR TITLE
PMM-10119: Implement displaying datapoint summaries

### PR DESCRIPTION
**Pull request by dliakhov**
What this PR does / why we need it:

Implements dislaying data point summaries for Telemetry in 'Advanced Settings' in the tooltip for 'Telemetry'

How it looks like:
image

Which issue(s) this PR fixes:
It was done while implementing [PMM-10119](https://jira.percona.com/browse/PMM-10119)

Fixes #

Special notes for your reviewer:

Feature build: https://github.com/Percona-Lab/pmm-submodules/pull/2688

### CODE REVIEW

File: public/app/percona/shared/components/Elements/LinkTooltip/LinkTooltip.tsx [hunk 1]
```diff
@@ -3,7 +3,8 @@ import { Icon, IconName, Tooltip, useTheme } from '@grafana/ui';
 import { getStyles } from './LinkTooltip.styles';
 
 export interface LinkTooltipProps {
-  tooltipText: string;
+  tooltipText?: string;
+  tooltipComponent?: React.Component;
```
Comment (fabio-silva) [inline public/app/percona/shared/components/Elements/LinkTooltip/LinkTooltip.tsx:7]: We can use just one prop for this. Maybe rename it to `tooltipContent: React.ReactNode`, which can be either text or a component.
Then, on render:
`{typeof tooltipContent === 'string' ? <span>tooltipContent</span> : tooltipContent}`

**Review by fabio-silva — CHANGES_REQUESTED** [review]

File: public/app/percona/settings/components/Advanced/Advanced.styles.ts [hunk 2]
```diff
@@ -112,5 +117,8 @@ export const getStyles = ({ v1: { breakpoints, spacing, typography, border, colo
       margin: ${spacing.sm};
       fill: ${colors.linkExternal};
     `,
+    telemetryTooltip: css`
+      padding-left: 10px;
```
[the remark this answers is not in the document]
Comment (dliakhov) [reply public/app/percona/settings/components/Advanced/Advanced.styles.ts]: Got it

Updated

File: public/app/percona/settings/components/Advanced/Advanced.styles.ts [hunk 3]
```diff
@@ -21,6 +21,11 @@ export const getStyles = ({ v1: { breakpoints, spacing, typography, border, colo
       display: flex;
       width: 230px;
     `,
+    wideAdvancedCol: css`
```
[the remark this answers is not in the document]
Comment (dliakhov) [reply public/app/percona/settings/components/Advanced/Advanced.styles.ts]: Removed

File: public/app/percona/settings/components/Advanced/Advanced.tsx [hunk 4]
```diff
@@ -163,6 +166,20 @@ export const Advanced: FC = () => {
   };
   const { Form } = withTypes<AdvancedFormProps>();
 
+  const getTooltipWithDatapointSummaries = () => {
```
[the remark this answers is not in the document]
Comment (dliakhov) [reply public/app/percona/settings/components/Advanced/Advanced.tsx]: Makes sense

Done

File: public/app/percona/shared/components/Elements/LinkTooltip/LinkTooltip.tsx (hunk 1 above)
Comment (dliakhov) [reply public/app/percona/shared/components/Elements/LinkTooltip/LinkTooltip.tsx]: Got it

Refactored

**Review by fabio-silva — APPROVED** [review]
Good job 👏

**Review by fabio-silva — APPROVED** [review]

**Review by fabio-silva — APPROVED** [review]

**Pull request merged**